docker/Makefile

```diff
@@ -1,5 +1,5 @@
 VERSION := 1.0.0
-ROOT_IMAGE ?= alpine:3.16
+ROOT_IMAGE ?= alpine:3.19.1
 CERT_IMAGE := $(ROOT_IMAGE)
 GOLANG_IMAGE := golang:1.22-alpine
```

Comment:
remove all 3 vars

Reply:
should I remove the arg declaration used in building the image too